src/main/java/net/minecraftforge/common/data/ForgeEntityTypeTagsProvider.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2021.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.common.data;
+
+import net.minecraft.data.DataGenerator;
+import net.minecraft.data.tags.EntityTypeTagsProvider;
+import net.minecraft.world.entity.EntityType;
+import net.minecraftforge.common.Tags;
+
+public class ForgeEntityTypeTagsProvider extends EntityTypeTagsProvider {
```

Comment:
Formatting: braces on newlines. Additionaly, to conform with its sibling tags providers, please mark this class as `final`.
```suggestion
public final class ForgeEntityTypeTagsProvider extends EntityTypeTagsProvider
{
```